wordpress-mobile/WordPress-iOS
# [Stats Refresh] add structure for stats detail list view

Fixes #n/a

This add the table structure to display stats' detail list views.

- When `View more` is selected on a stat card, the new `SiteStatsDetailTableViewController` is displayed.
  - Note: Custom `View more` are _not_ included (i.e. Latest Post Summary and Posting Activity.)
- For any `View more` selection, temporary data is displayed for now.
- On the list view:
  - The view title matches the selected stat title.
  - The column headers match the selected stat column headers.
  
Note: there's an extra line below the detail list view. That is specific to each stat type configuration. I'll address that when I add support for each type.

To test:

--- 
- On any applicable stat card, select `View more`.
- Verify:
  - The title is the stat type.
  - The column headers are specific to the stat type.
  - There are 3 fake data rows.

![detail_labels](https://user-images.githubusercontent.com/1816888/53271865-ec67a100-36ac-11e9-9364-f12386205746.png)
